fixtures/mockData.tsx

```diff
@@ -0,0 +1,46 @@
+const date = new Date();
+const currentDate = date.toLocaleDateString("nl-NL", {
+  year: "numeric",
+  month: "short",
+  day: "numeric",
+});
+
+const dateActive = date.toLocaleDateString("nl-NL", {
+  year: "numeric",
+  month: "short",
+  day: "numeric",
+  hour: "numeric",
+  minute: "numeric",
+});
```

Comment:
Ik denk dat je dit in je component wil hebben, niet in de fixtures. In je fixtures wil je echt alleen mock data hebben.